### DISCUSSION TITLE
Search bar

### DISCUSSION THREAD

**Discussion by GreenBioTheGreat** [lawand-dot-io/taskbar#59] [Feature requests]
There should be a feature in preferences where you can toggle between a fully functional search bar which also matches the feature suggestion for the **start menu**. Search bar can search through files from finder, and on the multi-media web browsers. [e.g _Safari, Chrome, Edge_] As I love the design for the TaskBar for macOS, this needs to be added.

**Comment by woweya** [comment]
+1 I support this 😭

**Comment by lawand-dot-io** [comment]
More context: https://github.com/lawand-dot-io/taskbar/discussions/95
